package.json

```diff
@@ -35,16 +33,15 @@
     "@mittwald/kubernetes": "^3.2.1",
     "config": "^1.30.0",
     "debug": "^3.1.0",
-    "elasticsearch": "^15.5.0",
     "email-addresses": "^3.0.3",
     "express": "^4.17.1",
     "google-protobuf": "^3.6.0",
     "grpc": "^1.24.3",
```

Comment:
Are `grpc`, `express` and `protobuf` still required?

Reply:
`express` yes (for the monitoring and health check server), the others not. Good catch.

Reply:
Fixed